src/pipeline_migration/migrate.py

```diff
@@ -282,8 +282,11 @@ def _apply_migration(pipeline_file: FilePath, migration: TaskBundleMigration) ->
 
         with resolve_pipeline(pipeline_file) as file_path:
             logger.info("Executing migration script %s on %s", migration_file, file_path)
+            cmd = ["bash", migration_file, file_path]
+            logger.debug("Run: %r", cmd)
             try:
-                subprocess.run(["bash", migration_file, file_path], check=True)
+                proc = sp.run(cmd, check=True, stderr=sp.STDOUT, capture_output=True)
+                logger.debug("%s", proc.stdout)
```

Comment:
If the script returns a non-0 exit code, we won't get the logs. Should we set `check=False` and then call `check_returncode()` after logging the output?

Reply:
> If the script returns a non-0 exit code, we won't get the logs.

Yeah, this is the behavior in my mind. But, I'll update the code to behave as what you said.

Reply:
Updated.